repobee_sanitizer/_sanitize_repo.py

```diff
@@ -128,9 +162,28 @@ def _sanitize_files(
             LOGGER.info(f"Shredded file {relpath}")
 
 
+def _format_error_string(files_with_errors: [_FilesWithErrors]) -> str:
+    """Takes a list of named tuples containing a files name, and a list of
+    errors and what lines they are found on. The list is then converted into a
+    properly formatted string to be printed on the command line.
+    """
+    formated_str = [
+        f"Syntax errors detected in {len(files_with_errors)} file(s):"
+    ]
+    for file in files_with_errors:
+        formated_str.append(f"\n{file.file_rel_path}")
+        for error in file.Error:
+            linestr = f"Line {error.line}: "
+            formated_str.append(
+                f"    {linestr if error.line else ''}{error.msg}"
+            )
+
+    return "\n".join(formated_str)
+
+
 def _sanitize_to_target_branch(
     repo_path: pathlib.Path, target_branch: str,
-) -> None:
+) -> Optional[_FilesWithErrors]:
```

Comment:
This hint is wrong (should be an optional _list_), but I would suggest removing the optional and just always returning a list.

```suggestion
) -> List[_FileWithErrors]:
```

Reply:
Just put `return []` at the end of the function.